src/pages/blog/categories/[category].js

```diff
@@ -0,0 +1,119 @@
+import { DarkModeToggle } from '../../../components/DarkModeToggle'
+
+const BlogCategoryPage = ({
+    data: {
+        allMarkdownRemark: { edges },
+        allMdx,
+    },
+    params: { category },
+}) => {
+    // Standardize position of 'slug' across MDX and MD
+    console.log(edges)
+    console.log(allMdx)
+
+    const parsedMdxData = allMdx.edges.map((edge) => ({
+        ...edge,
+        node: { ...edge.node, fields: { slug: `/${edge.node.slug}` } },
+    }))
+
+    const posts = [...edges, ...parsedMdxData] // Merge MDX and MD data into one array
+        .filter((edge) => {
+            const categories = edge.node.frontmatter.categories || ''
+            return !!edge.node.frontmatter.date && categories.indexOf(category) >= 0
+        })
+        .sort((a, b) => new Date(b.node.frontmatter.date) - new Date(a.node.frontmatter.date)) // Resort based on dates following merge
+
+    console.log(posts)
+    const latestPost = <PostCard key={posts[0].node.id} post={posts[0].node} featured={true} />
+    const nonLatestPosts = posts.slice(1, -1).map((edge) => <PostCard key={edge.node.id} post={edge.node} />)
+
+    const title = BlogCategories.find((k) => k.slug === category)?.title
+
+    return (
+        <div className="bg-offwhite-purple text-gray-900 dark:bg-darkmode-purple dark:text-white">
+            <Layout headerBackgroundTransparent={true}>
+                <SEO title="PostHog Blog" description="What we are up to, every week." />
+
+                <div className="bg-offwhite-purple text-gray-900 dark:bg-darkmode-purple dark:text-white">
+                    <div className="w-11/12 mx-auto text-right">
+                        <DarkModeToggle />
+                    </div>
+
+                    <Structure.Section className="my-12" width="5xl">
+                        <Structure.SectionHeader title={`Blog: ${title}`} titleTag="h1" titleClassName="text-center" />
+
+                        {latestPost}
+                        <NewsletterForm
+                            compact={true}
+                            className="bg-offwhite-purple dark:bg-darkmode-purple text-gray-900 dark:text-white"
+                        />
+
+                        <div className="w-11/12 max-w-3xl mx-auto flex flex-col lg:flex-row justify-between items-start">
+                            <div className="hidden lg:block lg:w-1/4 lg:pr-8">
+                                <BlogSidebar />
+                            </div>
+                            <div className="w-full lg:w-3/4 lg:pl-8">
+                                <header className="text-xs text-gray-400 uppercase">Recent Posts</header>
+                                {nonLatestPosts}
+                            </div>
+                        </div>
+                    </Structure.Section>
+                </div>
+            </Layout>
+        </div>
+    )
+}
+
+export default BlogCategoryPage
+
+export const pageQuery = graphql`
+    query {
```

Comment:
I think this query is pulling a lot of stuff we don't need/use

Reply:
Hm, not sure I follow. What things do you see that aren't being used? 

fields -> slug : used for link
id : used for key
excerpt(pruneLength: 250) : used for postcard excerpt
frontmatter -> date(formatString: "MMMM DD, YYYY") : used for resorting when merging mdx/md
frontmatter -> title : used in post card
frontmatter -> rootPage : used in query itself
frontmatter -> categories : used to filter posts

Reply:
Yeah sorry didn't properly follow the flow of props when reviewing in GH - looks good!